src/DataProvider.php

```diff
@@ -41,7 +41,7 @@ public static function getSubscribedEvents(): array
 
     public function onKernelRequest(RequestEvent $event): void
     {
-        if ($event->isMasterRequest()) {
+        if ($event->isMainRequest()) {
```

Comment:
This change will render the bundle incompatible with Symfony versions before 5.3. We could make this more dynamic or release a new major version with this change as-is and bump dependency versions 

cc @cindreta

Reply:
@zoka123 Ok, this makes obiously sense. I added a check for the existance of those methods. What do you think?

https://github.com/Treblle/treblle-symfony/pull/5/commits/f32041f73b596d4d1263a623c5c8a0f89bfe4462

Reply:
Oh, I just realised, the code had some missing parenthesis.
https://github.com/Treblle/treblle-symfony/pull/5/commits/a1ab997578518601cee1404c54d099cb5ebb5205